numpyro/contrib/autoguide/__init__.py

```diff
@@ -225,7 +230,8 @@ def get_transform(self, params):
         :return: the transform of posterior distribution
         :rtype: :class:`~numpyro.distributions.constraints.Transform`
         """
-        return handlers.substitute(self._get_transform, params)()
+        return ComposeTransform([handlers.substitute(self._get_transform, params)(),
```

Comment:
Are there any places where we are using `guide.unpack_latent` anymore if `get_transform` does the unpacking by default? If that is not the case, can we make this private `_unpack_latent` instead?

Reply:
Yes, we don't need those `unpack_latent` anymore! Nice catch!